engine/model.go

```diff
@@ -85,9 +85,40 @@ func (a *Address) Equals(o Address) bool {
 	return a.Network == o.Network && a.Address == o.Address
 }
 
+// Sets up OCSP stapling
+type OCSPSettings struct {
+	Enabled            bool
+	Period             string
+	Responders         []string // optional responder
+	SkipSignatureCheck bool
+}
+
+func (o *OCSPSettings) RefreshPeriod() (time.Duration, error) {
+	if o.Period == "" {
+		return time.Hour, nil
+	}
+	return time.ParseDuration(o.Period)
+}
+
+func (o *OCSPSettings) Equals(other *OCSPSettings) bool {
+	if o.Period != other.Period {
```

Comment:
What about o.Period == "1m", other.Period == "60s"? If you want that to compare as equal you could use `RefreshPeriod()` instead.